docs/how_to/index/modules.md

```diff
@@ -0,0 +1,15 @@
+# Modules
+
+```{toctree}
+---
+maxdepth: 1
+---
+vector_store_guide.ipynb
+List Index </guides/primer/index_guide.md>
```

Comment:
think you can link to the specific sections with "index_guide.md#list-index" etc.

Reply:
Oh nice.

Reply:
Hmm tried it quickly and doesn't seem to work.

Reply:
you need to use relative links iirc